backend/routes/portfolios.js

```diff
@@ -585,8 +616,8 @@ const additionalModelFactors = async (tickers, valuePredict, companyArrays) => {
 };
 
 const determineFactor = (averageAnalystRating) => {
-  const factorRange = 0.006;
-  const dilution = -(averageAnalystRating - 2) / 4;
+  const factorRange = 0.008;
+  const dilution = (3 - averageAnalystRating) / 4;
```

Comment:
nit: why not just factor the 4 into `factorRange` to begin with? If these are all kind of arbitrary values, I feel like you might as well not break the computation into unnecessary extra steps.

Like, isn't it the same if `factorRange` is 0.002 and `dilution` is `(3 - averageAnalystRating)`? I guess then `dilution` wouldn't be the right name for the variable (maybe `scalingFactor`?), but IMO the logic gets a little clearer.